src/replace_stl_math.sh

```diff
@@ -0,0 +1,14 @@
+for f in *.{cpp,h} ; do
```

Comment:
Did you mean to add this file?

Reply:
Yes, to catch STL instances in branches unmerged so far.  We can delete it in the future.